app/views/PartTimeNormalHoursView.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import pages.PartTimeNormalHoursPage
+
+@this(
+        govukLayout: templates.GovukLayoutWrapper,
+        formHelper: FormWithCSRF,
+        errorSummary: components.errorSummary,
+        button: components.button,
+        h1: components.h1,
+        p: components.p,
+        inputNumber: components.inputNumber
+)
+
+@(form: Form[_], period: Periods, idx: Int)(implicit request: Request[_], messages: Messages)
+
+@titleAndHeadingArgs = @{period match {
+    case fp: FullPeriod => Seq(dateToStringWithoutYear(fp.period.start), dateToString(fp.period.end))
+    case pp: PartialPeriod => Seq(dateToStringWithoutYear(pp.partial.start), dateToString(pp.partial.end))
+}}
+
+@govukLayout(pageTitle = Some(title(form, "partTimeNormalHours.title", titleMessageArgs = titleAndHeadingArgs)), thisPage = PartTimeNormalHoursPage) {
+
+    @formHelper(action = PartTimeNormalHoursController.onSubmit(idx)) {
+
+        @errorSummary(form.errors)
+
+        @h1(messages("partTimeNormalHours.heading", titleAndHeadingArgs: _*))
+
+        @if(period.isInstanceOf[PartialPeriod]) {
+            @p(Html(messages("partTimeNormalHours.partial", dateToStringWithoutYear(period.period.start), dateToString(period.period.end))))
+        }
+
+        @p{<span class="govuk-hint">@messages("partTimeNormalHours.p1")</span>}
```

Comment:
I suspect this would be better as a `p` with the correct class, rather than a `p` wrapping a `span`.